### ISSUE TITLE
Getting error when parsing swagger file 'API description parser warning in'

### ISSUE THREAD

**Issue by DFaiz** [apiaryio/dredd#1451]
**Describe the bug**
I've recently upgraded Dredd from version 4.8 to most recent, 11.2.15, and started getting the dozens of theses errors when Dredd attempts to parse the Swagger and generate the transaction request body. Each error is about a different row in the swagger file.

`warn: API description parser warning in /builds/swagger.yaml:64 (from line 64 column 8 to line 66 column 6): Unable to generate application/json; charset=utf-8 example message body out of JSON Schema`

This is the general settings of the swagger file:

```
swagger: '2.0'
info:
  version: '3'
  title: 'API'
schemes:
- 'http'
consumes:
- application/json; charset=utf-8
produces:
- application/json; charset=utf-8
```

The swagger file did not change between Dredd version upgrades. Please advise on what can be the cause.

**Expected behavior**
Generate transactions as usual.

**What is in your `dredd.yml`?**
Unfortunately, I'm unable to share the swagger file. But Dredd does manage to generate the request headers just fine.

**What's your `dredd --version` output?**
```
dredd v11.2.15 (Darwin 18.7.0; x64)
```

**Does `dredd --loglevel=debug` uncover something?**
Nothing, here the output:

```
Default value for a required parameter doesn't make sense from API description perspective. Use example value instead.
2019-08-06T09:32:48.297Z - debug: Starting the transaction runner
2019-08-06T09:32:48.297Z - debug: Starting reporters and waiting until all of them are ready
2019-08-06T09:32:48.298Z - debug: Beginning Dredd testing...
2019-08-06T09:32:48.298Z - debug: Sorting HTTP transactions
2019-08-06T09:32:48.298Z - debug: Configuring HTTP transactions
2019-08-06T09:32:48.303Z - debug: Reading hook files and registering hooks
2019-08-06T09:32:48.304Z - debug: Found Hookfiles: 0=/builds/hooks.js
2019-08-06T09:32:50.173Z - debug: Executing HTTP transactions
2019-08-06T09:32:50.173Z - debug: Running 'beforeAll' hooks
2019-08-06T09:32:50.174Z - debug: Running hooks...
hook: setting prime query
hook: setting prime query
2019-08-06T09:32:50.184Z - debug: Processing transaction #1: authorization > /authorizations > authorize request > 201 > application/json; charset=utf-8
2019-08-06T09:32:50.184Z - debug: Running 'beforeEach' hooks
2019-08-06T09:32:50.184Z - debug: Running hooks...
```

**Comment by honzajavorek** [comment]
@DFaiz Thanks for the issue. It is not apparent to me what could be the cause from the description. Could you please try to:

- assemble a minimal Swagger file which is still able to reproduce the issue, but doesn't contain any confidential info?
- figure out which version exactly within the range started to fail?

When figuring out the version, to save time, always [divide the interval](https://en.wikipedia.org/wiki/Divide-and-conquer_algorithm) in the middle and try installing version from the middle. If it is okay, you know that all versions up to this one is okay (if it fails, you know the problem is in that first half of the version range). Then repeat on the remaining interval until you get the version where it started.

**Comment by DFaiz** [comment]
@honzajavorek , I was contacted by @kylef , I will the support team the complete swagger file.

**Comment by DFaiz** [comment]
Was notified that this is indeed a bug in Dredd parser.

**Comment by kylef** [comment]
We've released an updated version of our parser in https://github.com/apiaryio/api-elements.js/releases/tag/fury-adapter-swagger%400.27.2. The next step is releasing an update to [dredd-transactions](https://github.com/apiaryio/dredd-transactions) to update the parser, then to updated Dredd itself to use the newer version of dredd-transactions.

**Comment by kylef** [comment]
@DFaiz This should be solved in Dredd 11.2.17, can you please give that a go.

**Comment by honzajavorek** [comment]
@DFaiz Feel free to let us know if the problem persists, I'm closing this now. Thanks for filing the issue!
